emulated_hue/__init__.py

```diff
@@ -21,11 +22,13 @@ def __init__(
         hass_token: str,
         http_port: int,
         https_port: int,
-    ):
+    ) -> None:
         """Create an instance of HueEmulator."""
         self._loop = None
         self._config = Config(self, data_path, http_port, https_port)
-        self._hass = HomeAssistant(url=hass_url, token=hass_token)
+        self._hass: Optional[HomeAssistantClient] = None
```

Comment:
@marcelveldt 
Any reason for doing it like this? Seems like this creates a bug in the latest version as _hass is not defined in __init__ so when `import HueApi` is called, None gets returned causing communication with hass to fail.